subsys/bluetooth/controller/ll_sw/lll_adv.h

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "pdu.h"
+
```

Comment:
Do not include in header file, include in c files that need it.

```suggestion
```

Reply:
The `PDU_ACCESS_ADDR_SIZE` and `PDU_CRC_SIZE` constants are used in this header file, in the definition of `struct lll_adv_sync`. Do you suggest that literals should be used in their stead?